contracts/src/lib/LibSubnetActor.sol

```diff
@@ -98,16 +101,16 @@ library LibSubnetActor {
 
             LibStaking.setMetadataWithConfirm(validators[i], publicKeys[i]);
             LibStaking.setFederatedPowerWithConfirm(validators[i], powers[i]);
-
-            s.genesisValidators.push(Validator({addr: validators[i], weight: powers[i], metadata: publicKeys[i]}));
+            s.genesis.addValidator(validators[i]);
```

Comment:
Is there are "remove" variant?

Reply:
Okay, found it at https://github.com/consensus-shipyard/ipc/pull/778/files#diff-db15c5a391bbc67defc15a7661eed42bf8488188847a6c72b0aa042e995c65baR206 

Not sure why it's in a different Solidity file, but it exists :+1: